packages/surveys/src/types.ts

```diff
@@ -0,0 +1,35 @@
+/* Copyright (c) 2020 Environmental Systems Research Institute, Inc.
+ * Apache-2.0 */
+
+import { IRequestOptions } from "@esri/arcgis-rest-request";
+import { IItem } from "@esri/arcgis-rest-types";
+
+type IResultsAvailability = "after" | "always";
```

Comment:
I think we should start documenting types. This would help developers know the purpose and design intent of a particular type. Particularly since this is for `SurveyResults` - maybe a prefix would also be useful.